### ISSUE TITLE
Anorectal stricture

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6448]
**Preferred term label:**
Anorectal stricture

**Synonyms**
Stricture of the anorectum

**Definition (free text, please give PubMed ID)**
Narrowing of the anorectum associated with inflammation or scar tissue.
DOI: doi:10.1111/apt.12894

**Parent term (use hpo.jax.org/app)**
Anorectal anomaly

**Comment by pnrobinson** [comment]
added, thanks
